js/scenes/Item.js

```diff
@@ -53,3 +54,16 @@ const styles = StyleSheet.create({
     ...defaultTextStyle,
   },
 });
+
+const mapStateToProps = () => {
+  return {};
+};
+
+const mapDispatchToProps = () => {
+  return {};
+};
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(Item);
```

Comment:
I followed the recommendations of [Redux about testing](http://redux.js.org/docs/recipes/WritingTests.html). I tried first the approach you suggested but it means that any components (bet it scenes or minor UI elements) must be provided with a store in every test file. I found it ended up adding too much code.